migration_utils/src/main/kotlin/com/pubnub/migration/App.kt

```diff
@@ -0,0 +1,93 @@
+/*
+ * This source file was generated by the Gradle 'init' task
+ */
+package com.pubnub.migration
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.core.Context
+import com.github.ajalt.clikt.core.main
+import com.github.ajalt.clikt.parameters.arguments.argument
+import com.github.ajalt.clikt.parameters.options.flag
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.types.path
+
+class Migration(private val replacements: List<Pair<String, String>>) : CliktCommand("migration_utils") {
```

Comment:
Would it be valuable to add info how to execute it or it is described somewhere else?

e.g. 
java -jar migration_utils.jar <project_directory> [options]
java -jar migration_utils.jar /path/to/my_project --dry-run --no-backups

Reply:
this will be in the documentation, actually there will be a file that does that for you so just
./migration_guide